r1cs/gadgets/crypto/src/crh/bowe_hopwood/mod.rs

```
@@ -63,10 +63,19 @@ where
                 input_in_bits.push(Boolean::constant(false));
             }
         }
-        assert!(input_in_bits.len() % CHUNK_SIZE == 0);
-        assert_eq!(parameters.params.generators.len(), W::NUM_WINDOWS);
+        // TODO: check error message
```

Let's return the same error message we returned for the primitive